Static/DataSource.swift

```diff
@@ -63,6 +63,16 @@ public class DataSource: NSObject {
     }
 
 
+	// MARK: - Public
+
+	public func rowAtPoint(point: CGPoint) -> Row? {
+		if let indexPath = tableView?.indexPathForRowAtPoint(point), row = rowForIndexPath(indexPath) {
```

Comment:
Tiny nit, but what would you think about just `return rowForIndexPath(indexPath)`, since that returns an optional anyway?

Shouldn't affect the output, just less code. :smile: